### DISCUSSION TITLE
PIH gridpoints off significantly.

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#593] [Q&A] [answered]
was doing some checking of various locations to make sure the GPS coordinates were indeed in the grid polygons, (except for those listed on the updates tab...) and noticed PIH seemed to be off by many gridpoints, averaging 11 miles...

is this a known issue?

![image](https://user-images.githubusercontent.com/12521900/202830173-f0190677-b9ee-46b7-9456-0b74a8f886a0.png)

**Comment by sullynole** [comment, marked as answer]
Nice sleuthing, this issue popped up on 11/16 and has been corrected.  PIH is now on the list of offices off by 1,1 (known issue).  For a faster response in the future, this type of operational issue can be reported via nco.ops@noaa.gov.
